src/seedu/addressbook/data/person/AddressConsists/PostalCode.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person.AddressConsists;
+
+/**
+ * Represents the postal code of a person's address.
+ */
+public class PostalCode {
+
+    private String value;
```

Comment:
I think you can implement a method to validate if the postal code is in the correct format, e.g. numbers.